stock_inventory_valuation_show_remaining/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+When you trigger a report of inventory valuation the total quantity and the
+total value of the layers show. This is confusing as users expect is the
```

Comment:
```suggestion
total value of the layers are displayed. This is confusing as users expect is the
```

Reply:
Thanks! applied that in the last commit.